riva/clients/asr/riva_asr_client.cc

```diff
@@ -78,6 +78,9 @@ DEFINE_double(stop_threshold, -1., "Threshold value to determine when endpoint d
 DEFINE_double(
     stop_threshold_eou, -1.,
     "Threshold value for likelihood of blanks before detecting end of utterance");
+DEFINE_string(
+    custom_configuration, "",
+    "Add custom configurations to be sent to the custom backends. <key:value,key:value>");
```

Comment:
slight modification in the text:
"Custom configurations to be sent to the server as key value pairs, <key:value,key:value,...>"

Reply:
Updated